src/Cake.Common/Solution/Project/ProjectParser.cs

```diff
@@ -76,14 +76,18 @@ public ProjectParserResult Parse(FilePath projectPath)
                      .Elements(ProjectXElement.Configuration)
                      .Select(cfg => cfg.Value)
                      .FirstOrDefault()
+                 let platform = propertyGroup
+                     .Elements(ProjectXElement.Platform)
+                     .Select(cfg => cfg.Value)
+                     .FirstOrDefault()
+                let configPropertyGroups = project.Elements(ProjectXElement.PropertyGroup)
```

Comment:
@wwwlicious very small comment, but... Can you make this let line up with the one above it? :smile: